### PR TITLE
Remove the ssr external for vue

### PR DESCRIPTION
**Pull request by kalepail**
## Changes

- What does this change?
Solves https://github.com/withastro/adapters/issues/207

## Testing

Tested the CF Pages deploy and it works now.

### CODE REVIEW

**Comment by alexanderniebuhr** [comment]
I'm not 100% sure where this should be landed. Either here or in the adapters PR 🤔 
Any other opinions?

File: packages/integrations/vue/src/index.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ async function getViteConfiguration(options?: Options): Promise<UserConfig> {
 		},
 		plugins: [vue(vueOptions), virtualAppEntrypoint(vueOptions)],
 		ssr: {
-			external: ['@vue/server-renderer'],
+			// external: ['@vue/server-renderer'],
```
Comment (ematipico) [inline packages/integrations/vue/src/index.ts:119]: This should be a removed line, not a comment

**Comment by ematipico** [comment]
> I'm not 100% sure where this should be landed. Either here or in the adapters PR 🤔 Any other opinions?

Why was `'@vue/server-renderer'` there in the first place? That's the question we should answer first. For the time being, I would land the PR in the clouldflare adapter, and **investigate** other adapters before landing *this* PR.

**Comment by kalepail** [comment]
fwiw this package is deprecated anyway as it's now included natively in the Vue package itself. (See https://www.npmjs.com/package/@vue/server-renderer)

**Review by ematipico — APPROVED** [review]
Looks good. @kalepail can you please create a changeset?

**Comment by kalepail** [comment]
@ematipico done...I think...first time committing a changeset

**Pull request merged**